src/improvements/tasks/MultisigTask.sol

```diff
@@ -1168,19 +1161,32 @@ abstract contract L2TaskBase is MultisigTask {
                 )
             );
         }
+    }
 
-        console.log("Parent multisig: ", address(parentMultisig_));
-
-        // This loads the allowed storage write accesses to storage for this task.
-        // If this task changes storage slots outside of the allowed write accesses,
-        // then the task will fail at runtime and the task developer will need to
-        // update the config to include the addresses whose storage slots changed,
-        // or figure out why the storage slots are being changed when they should not be.
-        for (uint256 i = 0; i < config.allowedStorageWriteAccesses.length; i++) {
+    /// @notice We use this function to add allowed storage accesses.
+    function _templateSetup(string memory) internal virtual override {
+        SuperchainAddressRegistry.ChainInfo[] memory chains = superchainAddrRegistry.getChains();
+        for (uint256 i = 0; i < config.allowedStorageKeys.length; i++) {
             for (uint256 j = 0; j < chains.length; j++) {
-                _allowedStorageAccesses.add(
-                    superchainAddrRegistry.getAddress(config.allowedStorageWriteAccesses[i], chains[j].chainId)
-                );
+                try superchainAddrRegistry.getAddress(config.allowedStorageKeys[i], chains[j].chainId) returns (
```

Comment:
we should make that remaining gas before the try/catches is over 500k or something for EIP-150 safety

Reply:
What do you think of this? https://github.com/ethereum-optimism/superchain-ops/pull/664/commits/b04046494b8b1b0e78635ca9d1e62ebc635d9b90